src/seedu/addressbook/data/person/Address.java

```diff
@@ -8,12 +8,17 @@
  */
 public class Address {
 
-    public static final String EXAMPLE = "123, some street";
-    public static final String MESSAGE_ADDRESS_CONSTRAINTS = "Person addresses can be in any format";
+	public static final int EXPECTED_SIZE_OF_ADDRESS_ARRAY = 4;
+    public static final String EXAMPLE = "a/123, Clementi Ave 3, #12-34, 231534";
+    public static final String MESSAGE_ADDRESS_CONSTRAINTS = "Person addresses must be in this format: a/BLOCK, STREET, UNIT, POSTAL_CODE";
     public static final String ADDRESS_VALIDATION_REGEX = ".+";
 
-    public final String value;
+    public String value;
```

Comment:
why is this still retained? you will introduce duplicity and possibly inconsistency due to this.